DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Imports:
     cli,
     DBI,
     dplyr,
+    framrosetta (>= 0.1.0),
```

Comment:
might get a little weird installing framrsquard if a dependency is another github repo

Reply:
I was worried about this for TAMMsupport, since it relies on my xldiff package. Seems to be just fine.